src/chesster.js

```diff
@@ -62,7 +61,7 @@ var chesster = new slack.Bot({
     config_file: config_file
 });
 
-function handleHeltourErrors(error){
+function handleHeltourErrors(bot, message, error){
```

Comment:
I know, right?I even caught a _different_ parameter error here, but not this one. LOL